docs/source/utils/vmimage.rst

```diff
@@ -0,0 +1,101 @@
+=======
+vmimage
+=======
+
+This utility provides a API to download/cache VM images (QCOW) from the
+official distributions repositories.
+
+Usage
+=====
+
+Import ``vmimage`` module::
+
+    >>> from avocado.utils import vmimage
+
+Get the image object, which will contains the path for the base image
```

Comment:
`will contains` looks odd to me. Also `of the` and not `for the`. How about: `which contains the path of the base image and the path of the external snapshot created for this instance.`

or simply: `Get any image which usually consist of a base image and an external snapshot`.